docs/apache-airflow/concepts.rst

```diff
@@ -770,35 +770,131 @@ passed, then a corresponding list of XCom values is returned.
     def pull_function(task_instance):
         value = task_instance.xcom_pull(task_ids='pushing_task')
 
-When specifying arguments that are part of the context, they will be
-automatically passed to the function.
-
 It is also possible to pull XCom directly in a template, here's an example
 of what this may look like:
 
-.. code-block:: jinja
+.. code-block:: python
+
+    "SELECT * FROM {{ task_instance.xcom_pull(task_ids='foo_task', key='table_name') }}"
+
+This can also be done using the ``output`` attribute of an operator. The above example can be then
+simplified:
+
+.. code-block:: python
+
+    f"SELECT * FROM { foo_task.output['table_name'] }"
```

Comment:
Should we document this ``output`` better somewhere?

Reply:
Yes, we should. I was to tired to write more about XComArgs